### DISCUSSION TITLE
Google Feedburner 提供的电子邮件订阅功能即将停止支持，更换电子邮件订阅为 follow.it

### DISCUSSION THREAD

**Discussion by DejavuMoe** [ppoffice/hexo-theme-icarus#875] [Ideas]
[从 7 月开始，我们正在将 FeedBurner 过渡到更稳定的现代基础架构。这将使所有用户的产品保持正常运行，但这也意味着我们届时将拒绝大多数非核心提要管理功能，包括电子邮件订阅。](https://support.google.com/feedburner/answer/10483501)

可以从 [FedBurner 无缝迁移到 follow.it ](https://follow.it/docs/publishers/getting-followers/how-can-i-switch-from-feedburner-to-follow-it)

**Comment by ppoffice** [comment]
@DejavuMoe 

https://github.com/ppoffice/hexo-theme-icarus/releases/tag/4.4.0
https://ppoffice.github.io/hexo-theme-icarus/Widgets/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E6%8C%82%E4%BB%B6/#follow-it

**Reply by DejavuMoe** [reply]
Great Work，Thank you!😁👍
